whosonfirst/feature.js

```diff
@@ -50,7 +50,14 @@ feature.getLocalLanguages = (feat, type = 'official') => {
 // see: https://github.com/whosonfirst-data/whosonfirst-data/issues/2154
 feature.getPlacetypeLocal = (feat) => _.flatten(
   feature.getLocalLanguages(feat, 'official')
+    // When a localized placetype isn't in a latin char set then WOF often provides
+    // a transliteration we can includesin a parenthetical, like: مقاطعة (muhafazah)
+    .map(lang => `label:${lang}_latn_x_preferred_placetype`)
```

Comment:
This should look more like:

if `label:${lang}_latn_x_preferred_placetype` && `label:${lang}_x_preferred_placetype`:
    placetypeLocal = `label:${lang}_x_preferred_placetype` (`label:${lang}_latn_x_preferred_placetype`)

Reply:
Paging @missinglink for a coding assist as this JS pattern is less obvious to me.